requirements.txt

```diff
@@ -18,6 +18,8 @@
 # same modules should appear in the setup.py list as given below.
 
 flake8==3.8.3
+pytest
```

Comment:
Please hard-pin each required package to a specific version.  Leaving them without a version specification can result in unexpected errors, either when setting up a sandbox or when testing.  We have seen such failures several times.

Also, please add each required package to the setup.py file as well as here.  The requirements.txt file is used to install necessary packages for a virtual environment established in this directory.  The setup.py file is used to install necessary packages for a virtual environment established in the directory for another repository, such as the application.  The duplication is unfortunate, but we haven't found a way yet to eliminate it.  The version specifications used in setup.py files usually specify a minimum version, e.g., <package_name>>=<version_number>, rather than hard-pinning.